### DISCUSSION TITLE
rights question

### DISCUSSION THREAD

**Discussion by applesoff** [Kozea/Radicale#1662] [Q&A server]
I have 3 users and 3 collections in 
collection root
|-me
----personal
----share
|-colleagues
----other
----schedule
----share (symlink)
|-boss
----schedule (symlink)

How can I set the rights file so that I have read/write access to everything in all directories and colleagues and boss have read access only to share and schedule?

I thought something like this would work, but I have zero understanding of this:

[colleagues]
user: me
collection: {share and schedule}
permissions: RWrw

[colleagues]
user: colleagues
collection: {share}
permissions: r

[boss]
user: boss
collection: {schedule}
permissions: r

**Comment by pbiering** [comment]
This should work, first match wins, so place this restrictions above the default permissions. The_ rights_ section also has extensive debug capabilities, so you can check whether it works as you like.
